qubes-rpc/nautilus/Makefile

```diff
@@ -5,3 +5,4 @@ NAUTILUSPYEXTDIR ?= /usr/share/nautilus-python/extensions
 install:
 	install -d $(DESTDIR)$(NAUTILUSPYEXTDIR)
 	install -t $(DESTDIR)$(NAUTILUSPYEXTDIR) -m 0644 *.py
+	install -t $(DESTDIR)$(NAUTILUSPYEXTDIR) -m 0644 *.sh
```

Comment:
Putting shell script into nautilus _python_ extension dir seems like a bad idea. I'd use just `/usr/lib/qubes`, where other scripts are already.
Also, please make it executable (0755).